### ISSUE TITLE
Can't focus in "create new" area of workshop

### ISSUE THREAD

**Issue by blueshifter** [Facepunch/RustWorkshop#167]
Perhaps a setting I can't find is the problem but I cannot focus on anything other than far away in the opposite direction from the skinned (or unskinned) objects.

![20170414225448_1](https://cloud.githubusercontent.com/assets/27533733/25060327/3a5e6a94-2168-11e7-9641-eafa64b57b18.jpg)

**Comment by rareox** [comment]
I have the same issue, the workshop use to automatically focus, but it seems to be broken since last update.

**Comment by garrynewman** [comment]
By focus we mean depth of field right, not the UI?

**Comment by blueshifter** [comment]
Yes it's the d.o.f. that isn't adjusting properly. I dual boot win/mac on an imac using bootcamp. No problems when I went into windows, the issue seems to be only in mac, for me at least. Hope that is helpful Garry and thanks for the horribly awesome game!

John
